parlai/tasks/ccpe/agents.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+from parlai.core.teachers import FixedDialogTeacher
+from .build import build
+import os
+import json
+
+
+class CCPETeacher(FixedDialogTeacher):
```

Comment:
Could we also add a teacher (or a flag) where we can provide every turn of dialogue? I.e., instead of just having the ASSISTANT utterances be labels, have _every_ utterance be a label